src/components/SessionExpiry.jsx

```diff
@@ -11,7 +11,7 @@ const SessionExpiry = () => {
 
   const displayTime = (timeLeft) => {
     if(Math.floor(timeLeft % 60) < 10){
-      return Math.floor(timeLeft/60) + ":" + "0" + Math.floor(timeLeft % 60)
+      return Math.floor(timeLeft/60) + ":0" + Math.floor(timeLeft % 60)
```

Comment:
we cud try DRY, say,

```javascript
hh = Math.floor(timeLeft/60);
mm = `${Math.floor(timeLeft % 60)}`.padStart(2, "0");

return `${hh} :${mm}`;

```

and avoid conditions completely.

Reply:
Done.